workshop/content/20-create-project/300-structure.md

```diff
@@ -36,12 +36,11 @@ You'll see something like this:
 Let's have a quick look at `bin/cdk-workshop.ts`:
 
 ```ts
-import cdk = require('@aws-cdk/cdk');
+import cdk = require('@aws-cdk/core');
 import { CdkWorkshopStack } from '../lib/cdk-workshop-stack';
 
 const app = new cdk.App();
 new CdkWorkshopStack(app, 'CdkWorkshopStack');
-app.run();
```

Comment:
Should that be app.synth?

Reply:
No this is not needed in the TypeScript version of the CDK as it's implied instead. You can see this from the default template from a `[cdk init](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/lib/init-templates/app/typescript/bin/%25name%25.template.ts)`.